src/ukify/ukify.py

```diff
@@ -1394,9 +1400,22 @@ def config_example(self) -> tuple[Optional[str], Optional[str], Optional[str]]:
 
 def apply_config(namespace, filename=None):
     if filename is None:
-        filename = namespace.config
-    if filename is None:
-        return
+        if namespace.config:
+            # Config file specified, turn it into a pathlib.Path
+            filename = namespace.config
+            print(f'Using config file: {filename}')
+            if not filename.is_file():
+                raise FileNotFoundError(f'Set config file not found or not a file: {filename}')
+        else:
+            # If there is no config file provided, try to look for default configs and load the first one found.
+            for config_dir in DEFAULT_CONFIG_DIRS:
+                if (pathlib.Path(config_dir) / DEFAULT_CONFIG_FILE).is_file():
+                    filename = pathlib.Path(config_dir) / DEFAULT_CONFIG_FILE
+                    print(f'Found config file: {filename}')
+                    break
+            if filename is None:
+                # No config file specified or found, nothing to do.
+                return
```

Comment:
```python
for config_dir in DEFAULT_CONFIG_DIRS:
    filename = pathlib.Path(config_dir) / DEFAULT_CONFIG_FILE
    if not filename.is_file():
       continue
else:
    # No config file specified or found, nothing to do.
    return
```

Reply:
That's better, why didn't I come up with that?

But wait, I'd still reckon that it's better to let the user know which config file is found by the ukify so they can easily find whatever causes trouble if they forget about that file, what do you think?